.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 # tmp files
 cookie.jar
 tmp/
+dist/
```

Comment:
Currently we do recommend to checkin the generated test case for easier accountability.